engine/gamesys/src/gamesys/components/comp_collection_proxy.cpp

```diff
@@ -166,14 +175,47 @@ namespace dmGameSystem
             dmhash_t component_instance_id = dmGameObject::GetIdentifier(c->m_Instance);
             if (component_instance_id == instanceId && c->m_ComponentIndex == index)
             {
-                comp_url_hash = c->m_Resource->m_UrlHash;
+                comp_url_hash = dmHashString64(GetCollectionResorcePath(c));
```

Comment:
Now this function returns urlhash of the collection it refers to instead of resource path to the component. It is needed to be able to get dependencies for collection proxy.